pkgs/servers/openafs/1.8/default.nix

```diff
@@ -44,19 +47,23 @@ stdenv.mkDerivation {
 
     ./regen.sh
 
+    for i in $(grep -l -R '/usr/bin/file\b' .); do
+      echo "Patch /usr/bin/file in $i"
+      substituteInPlace $i --replace "/usr/bin/file" "${stdenv.lib.getBin file}/bin/file"
+    done
```

Comment:
We tend to ignore this error since it happens in all autotools setups and does not harm.